src/main/java/com/mnewt00/vulcandatabase/storage/SQLiteStorageProvider.java

```diff
@@ -0,0 +1,127 @@
+package com.mnewt00.vulcandatabase.storage;
+
+import com.google.common.collect.Lists;
+import com.mnewt00.vulcandatabase.Log;
+import com.mnewt00.vulcandatabase.VulcanDatabase;
+import lombok.Getter;
+import lombok.SneakyThrows;
+
+import java.io.File;
+import java.sql.*;
+import java.util.List;
+import java.util.UUID;
+
+public class SQLiteStorageProvider implements StorageProvider {
+
+    @Getter
+    private final Connection connection;
+
+    public SQLiteStorageProvider() {
+        Connection connection1;
+        File database = new File(VulcanDatabase.getInstance().getDataFolder(), "database.db");
+        try {
+            Class.forName("org.sqlite.JDBC");
+            connection1 = DriverManager.getConnection("jdbc:sqlite:" + database.getAbsolutePath());
+        } catch (SQLException | ClassNotFoundException e) {
+            connection1 = null;
```

Comment:
You shouldn't need to specify this as null if you declare it when you declare connection1